clang-tools-extra/docs/clang-tidy/checks/cppcoreguidelines/pro-bounds-avoid-unchecked-container-accesses.rst

```diff
@@ -0,0 +1,51 @@
+.. title:: clang-tidy - cppcoreguidelines-pro-bounds-avoid-unchecked-container-accesses
+
+cppcoreguidelines-pro-bounds-avoid-unchecked-container-accesses
+===============================================================
+
+Flags the unsafe ``operator[]``. Can suggests fixing it with ``at()`` or a
+user-provided function.
+
+For example the code
+
+.. code-block:: c++
+
+  std::vector<int> a;
+  int b = a[4];
+
+will generate a warning but 
```

Comment:
Since it's written "but", I expect the code below to NOT generate a warning, so I find this text confusing.